packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -448,63 +448,63 @@ export const guides = [
         title: "Authorization",
         subLinks: [
           {
-            title: "Implement authorization by grant type",
+            title: "Implement authz by grant type",
             guideName: "implement-grant-type",
           },
           {
-            title: "Create an Authorization Server",
+            title: "Create an authorization server",
             guideName: "customize-authz-server",
           },
           {
-            title: "Configure SSO for Native apps",
+            title: "Configure SSO for native apps",
             guideName: "configure-native-sso",
           },
           { title: "Request user consent", guideName: "request-user-consent" },
           {
-            title: "Configure Device Authorization Grant Flow",
+            title: "Configure the Device Authz grant flow",
             guideName: "device-authorization-grant",
           },
           {
-            title: "Client secret rotation and key management",
+            title: "Secret rotation and key management",
```

Comment:
It doesn't need to repeat the title of the article, and you had said it was really about rotating keys, so how about:

```suggestion
            title: "Rotate client secrets",
```

Reply:
I would rather keep both so if they're looking for key management - they can find that, too. It fits - even on my laptop with an even more reduced size browser.